src/test/unit/specific_feature_tests/skins.js

```diff
@@ -0,0 +1,36 @@
+/* jshint maxlen: 100 */
+/* global -$,
+    QUnit,
+    equal,
+    module,
+    notEqual,
+    test,
+    prepareUnitTestModule
+*/
+"use strict";
+
+module(
+    "skin-ui-overrides",
+    {
+        setup: function () {
+            prepareUnitTestModule({
+                options: {skin: 'basic'}
+            });
+        }
+    }
+);
+
+test("skin's iframeHtml is used preferentially", function () {
+    var wym = jQuery.wymeditors(0);
+    QUnit.expect(2);
+
+    // Ensure that the basic skin has a non-default iframeHtml, so that this
+    // test is meaningful
+    notEqual(
+        WYMeditor.DEFAULT_OPTIONS.iframeHtml,
```

Comment:
This is an example of what I asked you about. It is so easy to make this mistake and produce this bug.